### PR TITLE
Feature/93/tests for redux, modalContext, AppMain

### PR DESCRIPTION
**Pull request by LubomyrKashuba**
tests for redux modalContext AppMain

### CODE REVIEW

**Review by byavdoshniak — APPROVED** [review]

File: src/components/popup-dialog/PopupDialog.js [hunk 1]
```diff
@@ -38,8 +38,9 @@ const PopupDialog = ({ content, closeModal }) => {
 
   return (
     <Dialog
-      fullScreen={ (width < 600) }
-      maxWidth="xl" onClose={ onClose }
+      data-testid='popup'
+      fullScreen={ (width < 600) } maxWidth="xl"
```
Comment (rostyslaw27) [inline src/components/popup-dialog/PopupDialog.js:42]: You can add useBreakpoints and do this: fullScreen={ (size === 'mobile') }

File: src/containers/guest-home-page/login-form/LoginForm.js [hunk 2]
```diff
@@ -27,8 +27,8 @@ const LoginForm = ({ handleSubmit, handleChange, handleBlur, data, errors }) =>
       />
       
       <TextField 
-        InputProps={ passwordVisibility }
-        error={ Boolean(errors.password) } 
+        InputProps={ { ...passwordVisibility, 'data-testid': 'password-input' } }
```
Comment (rostyslaw27) [inline src/containers/guest-home-page/login-form/LoginForm.js:30]: getByLabelText

File: src/containers/layout/AppHeader.js [hunk 3]
```diff
@@ -36,7 +36,8 @@ const AppHeader = () => {
           </IconButton>
 
           <Button
-            onClick={ openLoginDialog } size='medium' sx={ style.loginButton }
+            data-testid='open-modal' onClick={ openLoginDialog } size='medium'
```
Comment (rostyslaw27) [inline src/containers/layout/AppHeader.js:39]: getByText

**Review by rostyslaw27 — CHANGES_REQUESTED** [review]

File: src/components/popup-dialog/PopupDialog.js (hunk 1 above)
Comment (LubomyrKashuba) [reply src/components/popup-dialog/PopupDialog.js]: done

File: src/containers/guest-home-page/login-form/LoginForm.js (hunk 2 above)
Comment (LubomyrKashuba) [reply src/containers/guest-home-page/login-form/LoginForm.js]: fixed

File: src/containers/layout/AppHeader.js (hunk 3 above)
Comment (LubomyrKashuba) [reply src/containers/layout/AppHeader.js]: fixed

**Review by maarekua — APPROVED** [review]

**Pull request merged**